{{cookiecutter.project_slug}}/.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -52,8 +52,7 @@ body:
       description: |-
         Can be found by
         ```sh
-        python -c "import {{cookiecutter.package_name}};
-                   print({{cookiecutter.package_name}}.__version__)"
+        python -c "import {{cookiecutter.package_name}}; print({{cookiecutter.package_name}}.__version__)"
```

Comment:
This was me loving wrapping, but then copy-and-paste wouldn't work 😭

Reply:
🎁